src/main/java/io/spring/pind/websock/controller/ChatRoomController.java

```diff
@@ -0,0 +1,47 @@
+package io.spring.pind.websock.controller;
+
+import io.spring.pind.websock.entity.ChatRoom;
+import io.spring.pind.websock.repository.ChatRoomRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RequiredArgsConstructor
+@Controller
+@RequestMapping("/chat")
+public class ChatRoomController {
+
+    private final ChatRoomRepository chatRoomRepository;
+
+    @GetMapping("/room")
+    public String rooms(Model model) {
+        return "/chat/room";
+    }
+
+    @GetMapping("/rooms")
+    @ResponseBody
+    public List<ChatRoom> room() {
+        return chatRoomRepository.findAllRoom();
+    }
```

Comment:
`/room`으로 요청 들어올 경우에 
단순히 response로 문자열 데이터만 넘겨주는 메소드로 보이는데
정확하게 어떤 기능을 하는지 잘 모르겠어서 설명 부탁드려도 될까요?
(`/room/enter/{roomId}` 요청도 설명 부탁드릴게요!)

그리고 각각의 메소드명만 보고서는 기능이 엇갈린? 느낌이 들어서 좀 더 상세하게 수정하는게 나을 것 같습니다!
```suggestion
    @GetMapping("/room")
    public String room(Model model) {
        return "/chat/room";
    }

    @GetMapping("/rooms")
    @ResponseBody
    public List<ChatRoom> getRoomList() {
        return chatRoomRepository.findAllRoom();
    }
```

Reply:
수정 완료했습니다!